devices/Seesaw/Seesaw.nuspec

```diff
@@ -20,7 +20,13 @@
     <summary>Iot.Device.Seesaw assembly for .NET nanoFramework C# projects</summary>
     <tags>nanoFramework C# csharp netmf netnf Iot.Device.Seesaw</tags>
     <dependencies>
-      <dependency id="nanoFramework.CoreLibrary" version="1.10.4-preview.11" />
+      <dependency id="nanoFramework.CoreLibrary" version="1.11.7" />
+      <dependency id="nanoFramework.Runtime.Events" version="1.9.2" />
```

Comment:
are all those the latest versions? and correctly in sync with the nfproj? Just not super easy to read in this mode.